src/website/typography/Heading/Heading.js

```diff
@@ -0,0 +1,54 @@
+export default function Heading({
+    typeStyle = "heading-lg",
```

Comment:
I'd remove the word `heading` from all the `typeStyle` options. It's a bit repetitive and unnecessary since we already know it's for heading because we're on the heading component :)